dotCMS/src/main/java/com/dotmarketing/cms/createaccount/action/CreateAccountAction.java

```diff
@@ -48,7 +51,8 @@
 import com.liferay.portal.model.Company;
 import com.liferay.portal.model.User;
 
-public class CreateAccountAction extends DispatchAction {
+@Deprecated
+public class CreateAccountAction extends SecureAction {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)